releasenotes/notes/curve-analysis-02a702a81e014adf.yaml

```diff
@@ -0,0 +1,51 @@
+---
+prelude: >
+    To be written.
+features:
+  - |
+    :class:`CurveAnalysis` delegates the core fitting functionality to the LMFIT package.
+    Analysis class author can flexibly define new analysis instance
+    with LMFIT ``Model`` objects. In addition, new analysis option ``fit_method`` is added
+    so that user can perform fit with several algorithms offered by the LMFIT minimizer.
+    See LMFIT documentation for user guide.
+  - |
+    New dataclass :class:`.CurveFitResult` is introduced. This dataclass includes
+    rich context of curve fitting status with several extra statistics.
+    This is a wrapper class of LMFIT ``MinimizerResult`` with some extention.
+    Fit parameters in UFloat representation are stored while keeping
+    correlation between them, which are accessible with the ``.ufloat_params`` attribute.
+    Note that the value of the first analysis result entry titled with ``@Parameters_*`` 
+    has been replaced with this dataclass. This entry has been just a list of fit values 
+    in Python float format with covariance matrix separately stored in ``.extra`` metadata.
+    New object provides users with a better experience for understanding the analysis outcome.
+  - |
+    New curve drawer option ``figure_title`` is added. This allows user to show
+    an arbitrary string in the output figure title. See the example code below
+    to learn how to set the option.
+    
+    .. code-block:: python
+    
+      exp = MyExperiment(...)
+      exp.analysis.drawer.set_options(figure_title="Qubit0")
+    
+    Note that ``.analysis`` should be a subclass of :class:`.BaseCurveAnalysis`.
+upgrade:
+  - |
+    :class:`DragCalAnalysis` can take more than three curves with different number of
```

Comment:
I think this point needs some more context, I dont understand what was changed just from reading it